### PR TITLE
Variant attribute values pagination

### PR DESCRIPTION
**Pull request by orzechdev**
I want to merge this change because... it adds variant attribute values pagination support, in the following views:
- variant details page
- variant create page
- variant creator (here views are not 1:1 with design, there are many things to update beside pagination support, so I think it's better to do it in other time)

**PR intended to be tested with API branch:** 

### Pull Request Checklist

1. [x] This code contains UI changes
2. [x] All visible strings are translated with proper context including data-formatting
3. [x] Attributes `[data-test-id]` are added for new elements
4. [ ] Changes are mentioned in the changelog
5. [x] The changes are tested in different browsers and in light/dark mode

### Test environment config

API_URI=https://saleor-3071-paginate-attributes.api.saleor.rocks/graphql/

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-3347 Add pagination to attribute values selection card in variant details pages](https://app.clickup.com/t/2549495/SALEOR-3347)

**Comment by patrys** [comment]
Task linked: [SALEOR-3109 Use pagination on attribute values](https://app.clickup.com/t/2549495/SALEOR-3109)

**Review by jwm0 — APPROVED** [review]

File: src/products/components/ProductVariantCreatorPage/ProductVariantCreatorPage.tsx [hunk 1]
```diff
@@ -28,8 +29,11 @@ const useStyles = makeStyles(
     },
     content: {
       overflowX: "visible",
-      overflowY: "hidden",
-      width: 800
+      [theme.breakpoints.up("md")]: {
+        position: "absolute",
+        width: `calc(100vw - ${drawerWidthExpanded + theme.spacing(6)}px)`,
```
Comment (dominik-zeglen) [inline src/products/components/ProductVariantCreatorPage/ProductVariantCreatorPage.tsx:34]:
```suggestion
        width: `calc(100vw - ${drawerWidthExpanded}px - ${theme.spacing(6)}px)`,
```
Makes it easier to switch spacing unit to rems in future

File: src/products/components/ProductVariantCreatorPage/ProductVariantCreatorPage.tsx [hunk 2]
```diff
@@ -28,8 +29,11 @@ const useStyles = makeStyles(
...
+        maxWidth: theme.breakpoints.width("lg") - theme.spacing(6)
```
Comment (dominik-zeglen) [inline src/products/components/ProductVariantCreatorPage/ProductVariantCreatorPage.tsx:35]: For the same reason as above, please use calc

File: src/products/components/ProductVariantCreatorPage/ProductVariantCreatorValues.tsx [hunk 3]
```diff
@@ -67,32 +114,24 @@ const ProductVariantCreatorValues: React.FC<ProductVariantCreatorValuesProps> =
...
+                name={`attribute:${attribute.name}`}
+                label={intl.formatMessage(messages.multipleValueLabel)}
+                value={getMultiValues(data.attributes, attribute)}
+                onChange={event =>
+                  handleValueClick(attribute.id, event.target.value)
+                }
+                allowCustomValues={true}
+                fetchChoices={fetchAttributeValues}
+                {...fetchMoreAttributeValues}
+                onClick={() => onAttributeSelect(attribute.id)}
```
Comment (dominik-zeglen) [inline src/products/components/ProductVariantCreatorPage/ProductVariantCreatorValues.tsx:133]: I get a feeling that it should be named `onFocus`

**Review by dominik-zeglen — CHANGES_REQUESTED** [review]

**Review by pwgryglak — CHANGES_REQUESTED** [review]
Looks good, but for example in attribute 0705TestVAtt, o na product level I can't get to values from 21 to 27 via dropdown. Plus each time the values load I am transferred to top of the list. It's poor UX

**Review by eCzerniak — APPROVED** [review]

**Review by pwgryglak — APPROVED** [review]
On Safari looks good, Still experiencing on Chrome but I experience issues on Chrome that nobody else does so - Approve

**Review by dominik-zeglen — APPROVED** [review]

**Pull request merged**